packages/reactive-element/src/reactive-element.ts

```diff
@@ -817,6 +821,9 @@ export abstract class ReactiveElement
   }
 
   /**
+   * Install a `ReactiveController` onto the element's reactive update cycle.
+   * LitElement automatically calls into any installed controllers during its
+   * lifecycle callbacks.
```

Comment:
"Install... onto" seems a little awkward? Maybe:

```suggestion
   * Registers a `ReactiveController` to participate in the element's reactive update cycle.
   * The element automatically calls into any registered controllers during its
   * lifecycle callbacks.
```
This matches the language in the guide more closely. Only thing I don't like is the term "registered controllers" which sounds kinda weird. 

For the second sentence, we could say, "`LitElement`" (the class) or "Lit" (the library) but in this case I thought maybe it was clearer to just say, "the element" because we're specifically talking about a single instance (this instance invokes its registered controllers).